File: wcserver/c#/examples/BSMiniRPTC#.NET/clsList.cs
using System;
using System.Collections;
using System.Diagnostics;
using System.Drawing;
using System.Windows.Forms;

namespace BSMiniRPT
{
	public class clsList
	{

		private string mvarName = "";
		private int mvarItemData = 0;

	#region Public Properties ...

		public string DisplayName
		{
			get
			{
				return mvarName;
			}
			set
			{
				mvarName = value;
			}
		}

		public int ItemData
		{
			get
			{
				return mvarItemData;
			}
			set
			{
				mvarItemData = value;
			}
		}

	#endregion

	#region Public Subroutines ...

		public clsList()
		{

		}

		public clsList(string DisplayName, int ItemData)
		{

			mvarName = DisplayName;
			mvarItemData = ItemData;

		}

	#endregion

	#region Public Functions ...

		public override string ToString()
		{

			return mvarName;

		}

	#endregion

	}

} //end of root namespace

File: wcserver/vb.net/examples/BSMiniRPT.NET/clsList.vb
Option Strict On
Option Explicit On

Public Class clsList

    Private mvarName As String = ""
    Private mvarItemData As Integer = 0

#Region "Public Properties ..."

    Public Property DisplayName() As String
        Get
            Return mvarName
        End Get
        Set(ByVal value As String)
            mvarName = value
        End Set
    End Property

    Public Property ItemData() As Integer
        Get
            Return mvarItemData
        End Get
        Set(ByVal value As Integer)
            mvarItemData = value
        End Set
    End Property

#End Region

#Region "Public Subroutines ..."

    Public Sub New()

    End Sub

    Public Sub New(ByVal DisplayName As String, ByVal ItemData As Integer)

        mvarName = DisplayName
        mvarItemData = ItemData

    End Sub

#End Region

#Region "Public Functions ..."

    Public Overrides Function ToString() As String

        Return mvarName

    End Function

#End Region

End Class
